docs/modules/ROOT/pages/faqs.adoc

```diff
@@ -1,83 +1,100 @@
 = Cassandra Source Connector FAQs
 
-If you are new to DataStax Cassandra Source Connector, these FAQs are for you.
+If you are new to DataStax Cassandra Source Connector (CSC), please read the following FAQs.
 
 == Introduction
 
-=== What is DataStax Cassandra Source Connector?
+=== What is DataStax CSC?
 
-The Alpha release of Cassandra Source Connector is a new, open-source product from DataStax.
-With CDC, Cassyour apps can consume data from an Apache Cassandra&reg; table and put the updates onto a Pulsar topic, which in turn can write the data to external targets such as Elastic, Snowflake, and other platforms. By design, the Source component is simple, with a 1:1 correspondence between the Cassandra table and a single Pulsar topic.
+The Alpha release of the Cassandra Source Connector (CSC) is a new, open-source product from DataStax.
 
-=== What are the system requirements for Cassandra Data Connector?
+With CDC, your apps can consume data from an Apache Cassandra table and submit the updates onto a Pulsar topic.  This topic can then be consumed by external targets such as Elastic, Snowflake, and other platforms. By design, the CSC is meant to be a simple connector with a 1:1 correspondence between a Cassandra table and a single Pulsar topic.
 
-Cassandra Source Connector requires additional resources depending on your write throughput.
-For each cassandra write, each Cassandra replica send one message to your streaming platform (Pulsar or Kafka), and
-execute at least one QUORUM read on your
+The CSC is composed of two parts:
 
-=== What components and features are provided by Cassandra Data Connector?
+* The Cassandra Source Connector, which transmits Change Data Capture (CDC) updates from the Cassandra table to the streaming service.
+* The Pulsar CSC Connector, which verifies the CSC events and stores the final data changes into a Data topic.
 
-Cassandra Data Connector provides:
+=== What are the system requirements for the CSC?
+
+The CSC requires additional resources depending on your write throughput.
```

Comment:
Can we provide more information here? Perhaps an example of what resources are required at the lower end and at the higher end. I don't walk away feeling like I am any clearer on what is actually required.

Reply:
Let's see what we can get.  @vroyer: Can we provide any additional details?  We're really nebulous on this.